openslides_backend/action/actions/meeting/create.py

```diff
@@ -67,6 +67,14 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 f"You cannot create a new meeting, because you reached your limit of {limit_of_meetings} active meetings."
             )
 
+        if (
+            instance.get("start_time")
+            and not instance.get("end_time")
+            or not instance.get("start_time")
+            and instance.get("end_time")
+        ):
+            raise ActionException("Only one of start_time and end_time is not allowed.")
```

Comment:
Move into a helper method in `MeetingPermissionMixin` (which then needs to be renamed to `MeetingMixin`) to avoid the code duplication.